server/app/services/applicant/ApplicantService.java

```diff
@@ -591,6 +592,21 @@ CompletionStage<ApplicationModel> submitApplication(
           Optional<StatusDefinitions.Status> maybeDefaultStatus =
               activeStatusDefinitions.getDefaultStatus();
 
+          Optional<Boolean> optionalEligibilityStatus =
```

Comment:
question (maybe blocking): Is there an `ApplicantService` test you can update to test that the eligibility is saved correctly here?